drivers/frequency/ad9517/ad9517.h

```diff
@@ -337,6 +343,13 @@ struct ad9517_lvpecl_channel_spec {
 	uint8_t name[16];	  // Optional descriptive channel name.
 };
 
+enum ad9517_type {
+	AD9517_1 = 0,
+	AD9517_2 = 1,
+	AD9517_3 = 2,
+	AD9517_4 = 3,
```

Comment:
Nitpick: extra , 
Also if you don't want specific values for an enum you can not initialize them. By default the will have have values from 0 to n

Reply:
I know. I just wanted to make it clear enough for the user. Will remove the explicit initialization.